src/lib/components/services/icons/no-update-needed.svelte

```diff
@@ -1,9 +1,14 @@
 <script>
   import { historyLineIcon } from "$lib/icons";
+
+  export let small = false;
+
+  const dimension = small ? "h-s16 w-s16" : "h-s24 w-s24";
+  const padding = small ? "p-s8" : "p-s12";
 </script>
 
-<div class="rounded-full bg-service-blue p-s12">
-  <div class="h-s24 w-s24 fill-current text-service-blue-dark">
+<div class={`rounded-full bg-service-blue ${padding}`}>
```

Comment:
```suggestion
<div class="rounded-full bg-service-blue {padding}">
```
je te laisse corriger les autres ;-)